packages/vite/src/node/plugins/reporter.ts

```diff
@@ -183,7 +183,6 @@ export function buildReporterPlugin(config: ResolvedConfig): Plugin {
             path.resolve(config.root, outDir ?? config.build.outDir),
           ),
         )
-        const assetsDir = `${config.build.assetsDir}/`
```

Comment:
Looks like the `assetsDir` printing logic was added in https://github.com/vitejs/vite/pull/10895. I'm fine with dimming folders by default but good to loop in @ArnaudBarre for his thoughts on this too.